solidity/contracts/isms/MultisigIsm.sol

```diff
@@ -99,10 +101,12 @@ contract MultisigIsm is IMultisigIsm, Ownable {
         uint32[] calldata _domains,
         address[][] calldata _validators
     ) external onlyOwner {
-        require(_domains.length == _validators.length, "!length");
-        for (uint256 i = 0; i < _domains.length; i += 1) {
+        uint256 domainsLength = _domains.length;
```

Comment:
prefix with `_` to match the rest of our style, here and everywhere else in this PR..